src/celeritas/em/model/GammaNuclearModel.cc

```diff
@@ -98,5 +111,49 @@ void GammaNuclearModel::step(CoreParams const&, CoreStateDevice&) const
 }
 #endif
 
+//---------------------------------------------------------------------------//
+/*!
+ * Build CHIPS gamma-nuclear element cross sections using G4GammaNuclearXS.
+ */
+inp::Grid
+GammaNuclearModel::calc_chips_xs(AtomicNumber z, double emin, double emax) const
+{
+    CELER_EXPECT(z);
+
+    inp::Grid result;
+
+    // Tabulate cross sections using separate parameterizations for the high
+    // energy region (emin < E < 50 GeV) and the ultra high energy region up
+    // to the maximum valid energy (emax). The numbers of bins are chosen to
+    // adequately capture both the parameterized points (224 bins from 106 MeV
+    // to 50 GeV) and the calculations used in G4PhotoNuclearCrossSection,
+    // which can be made configurable if needed (TODO). Note that the linear
+    // interpolation between the upper limit of the IAEA cross-section data
+    // and 150 MeV, as used in G4GammaNuclearXS, is also included in this
+    // tabulation.
+    double const emid = 5e+4;
```

Comment:
An extra comment for this variable (including units) might improve clarity.

Reply:
Added a breief description for emid.